modules/ROOT/pages/rtc/rtc-troubleshooting.adoc

```diff
@@ -56,7 +56,7 @@ To resolve this issue ensure all computer clocks are synchronised using https://
 
 === Unable to retrieve JWK set
 
-A "JWK set" is a structure that contains public keys for validating JWT signatures. At least one public key in the set must match the private key used to sign the provided identity token for RTC. This error means a public key could not be found in the {cloudname} account of the {companyname} API key used to load the RTC plugin. For information on creating public/private key pairs, see: link:{baseurl}/rtc/jwt-authentication/#privatepublickeypairsfortinycloudservices[JWT Authentication].
+A "JWK set" is a structure that contains public keys for validating JWT signatures. At least one public key in the set must match the private key used to sign the provided identity token for RTC. This error means a public key could not be found in the {cloudname} account of the {companyname} API key used to load the RTC plugin. For information on creating public/private key pairs, see: xref:rtc/jwt-authentication.adoc#privatepublickeypairsfortinycloudservices[JWT Authentication].
```

Comment:
This xref doesn't resolve as the `privatepublickeypairsfortinycloudservices` id doesn't exist. I'm not exactly sure where this is meant to link to though and is broken in production as well... this will probably be a good one to note down to fix later.

Reply:
comment added here and also in the other 2 broken links in this page

Reply:
ok, I only add a comment to this and the other link, I changed the last link as you suggested